main.go

```diff
@@ -29,9 +30,7 @@ func rssHandler(rw http.ResponseWriter, r *http.Request) {
 		Created:     time.Now(),
 	}
 
-	for _, feed := range sourceFeeds {
-		fetch(feed, master)
-	}
+	fetchFeeds(master)
```

Comment:
I didn't quite follow where each of these changes would go compared to the current solution but it sounds like it might be cleaner without the wait group?